android/src/main/java/com/vgscollectreactnative/CollectorManager.kt

```diff
@@ -26,6 +29,26 @@ class CollectorManager internal constructor(context: ReactApplicationContext?) :
     }
   }
 
+  @SuppressLint("ResourceType")
+  @ReactMethod
+  fun confirmPin(name: String, path: String, method: String, headers: ReadableMap, callback: Callback) {
```

Comment:
here, I would revisit parameters being used/needed for pinConirm. Also, I would use the method name for ios and android: here is `confirmPin` and ios is `pinConfirm`